### PR TITLE
Register huygd.is-a.dev subdomain

### PR DESCRIPTION
**Pull request by huygmdvn**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/huygd.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "My first domain to learn",
+  "repo": "https://github.com/huygmdvn/register",
+  "owner": {
+    "username": "huygmdvn",
+    "email": "huynhquochuytkna@gmail.com"
+  },
+  "record": {
+    "A": "103.77.240.59"
```
Comment (CuteDog5695) [inline domains/huygd.json:9]:
```suggestion
    "CNAME": "register-4ve.pages.dev"
```

File: domains/nsupdate.json
Comment (CuteDog5695) [inline domains/nsupdate.json:1]: Delete this since i dont think you have a use case for it.

**Review by iostpa — CHANGES_REQUESTED** [review]
Site seems to be blank, please add more content to it.

**Review by iostpa — APPROVED** [review]
Seems good to me now (gave website preview in discord)

**Pull request merged**